Src/GBX.NET.PAK/PakFile.cs

```diff
@@ -5,20 +5,22 @@ public sealed class PakFile
     public string Name { get; }
     public string FolderPath { get; }
     public uint ClassId { get; }
-    public int Offset { get; }
+    public uint Offset { get; }
     public int UncompressedSize { get; }
     public int CompressedSize { get; }
     public int? Size { get; }
     public UInt128? Checksum { get; }
     public ulong Flags { get; }
-
-    public bool IsCompressed => (Flags & 0x7C) != 0;
+    public bool PublicFile => (Flags & 0x2000000000000) != 0;
+    public bool ForceNoCrypt => (Flags & 0x4000000000000) != 0;
+    public bool IsCompressed => (Flags & 0x3C) != 0;
+    public bool IsEncrypted => !ForceNoCrypt && !PublicFile;
```

Comment:
Pls also add the `DontUseDummyWrite` from the PAK wiki and in the `OpenGbxFile`, check for this value and set the `EncryptionInitializer` only if it's false. Maybe rename this variable to something less confusing if you have an idea.